client/blocks/site-address-changer/index.jsx

```diff
@@ -214,11 +213,27 @@ export class SiteAddressChanger extends Component {
 		return currentDomainName.replace( currentDomainSuffix, '' );
 	}
 
+	/**
+	 * This is an edge case scenario where user have the site address changer opened and the user transfers
+	 * the site to atomic on other tab/window, losing sync between client and server. Client will try to
+	 * check availability against wordpress.com and will receive 404s because site is transfered to wpcomstaging.com
+	 */
+	isUnsyncedAtomicSite() {
+		const { validationError } = this.props;
+		const serverValidationErrorStatus = get( validationError, 'errorStatus' );
+
+		return serverValidationErrorStatus === 404;
```

Comment:
Is there an advantage to using `get` here? If `validationError` is sometimes undefined, we could do:

```suggestion
		return 404 === validationError?.errorStatus;
```

Reply:
Sounds good! It was the convention in the file, but avoiding lodash would be better.